src/status_im/contexts/shell/constants.cljs

```diff
@@ -2,30 +2,31 @@
 
 (def ^:const floating-shell-button-height 44)
 
-(def ^:const default-selected-stack :wallet-stack)
+(def ^:const default-selected-stack :screen/wallet-stack)
```

Comment:
The only side effect of this migration is that we store the last selected stack ID in async storage, which will be something like :communities-stack. Since this no longer exists, the selected stack ID will fall back to :screen/wallet-stack.